src/core/ext/filters/http/server/http_server_filter.h

```diff
@@ -26,4 +26,10 @@
 /* Processes metadata on the server side for HTTP2 transports */
 extern const grpc_channel_filter grpc_http_server_filter;
 
+namespace grpc_core {
+// Temporary code that allows servers to accept PUT requests. DO NOT USE WITHOUT
+// PERMISSION.
+void InternalOnlyAllowPutRequests();
```

Comment:
Just to be safe, let's call this something like `InternalOnlyDoNotUseUnlessYouHavePermissionFromGrpcTeamAllowPutRequests()`. :)

Reply:
...AllowBrokenPutRequests...

Broken being load bearing

Reply:
Done!